src/tasks/netcore/updateBlazorManifest.ts

```diff
@@ -0,0 +1,105 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as xml2js from 'xml2js';
+import ChildProcessProvider from "../../debugging/coreclr/ChildProcessProvider";
+import { DockerContainerVolume } from '../../debugging/coreclr/CliDockerClient';
+import LocalOSProvider from "../../debugging/coreclr/LocalOSProvider";
+import { OSTempFileProvider } from "../../debugging/coreclr/tempFileProvider";
+import { ext } from "../../extensionVariables";
+import { pathNormalize } from '../../utils/pathNormalize';
+import { PlatformOS } from '../../utils/platform';
+import { execAsync } from '../../utils/spawnAsync';
+import { DockerRunTaskDefinition } from "../DockerRunTaskProvider";
+import { DockerRunTaskContext } from "../TaskHelper";
+
+interface ContentRootAttributes {
+    BasePath: string;
+    Path: string;
+}
+
+interface ContentRoot {
+    $: ContentRootAttributes;
+}
+
+interface StaticWebAssets {
+    ContentRoot: ContentRoot[];
+}
+
+interface Manifest {
+    StaticWebAssets: StaticWebAssets;
+}
+
+export async function updateBlazorManifest(context: DockerRunTaskContext, runDefinition: DockerRunTaskDefinition): Promise<void> {
+    const tempFileProvider = new OSTempFileProvider(new LocalOSProvider(), new ChildProcessProvider());
+
+    const locationsFile = tempFileProvider.getTempFilename();
+
+    const targetsFile = path.join(ext.context.asAbsolutePath('resources'), 'GetBlazorManifestLocations.targets');
+
+    const command = `dotnet build /r:false /t:GetBlazorManifestLocations /p:CustomAfterMicrosoftCommonTargets=${targetsFile} /p:BlazorManifestLocationsOutput=${locationsFile} ${runDefinition.netCore.appProject}`;
```

Comment:
Is this going to be executed for every .NET Core project?

Reply:
Yes. The most we can do is restrict it to projects using the `Microsoft.NET.Sdk.Web` SDK, but I'm not entirely sure if that's an appropriate filter. We don't have any way of knowing more information about .NET Core projects.

Reply:
Then again, we do it only for AspNetCore in VS, so I'll filter on that SDK.